app/Services/Company/Adminland/Employee/ImportEmployeesFromTemporaryTable.php

```diff
@@ -33,10 +38,21 @@ public function rules(): array
      *
      * @param array $data
      */
-    public function execute(array $data): void
+    public function init(array $data): self
     {
         $this->data = $data;
         $this->validate();
```

Comment:
We call `validate()` twice. Here, and in the `execute()` method. I think it's a mistake, right?

Reply:
The validate is run on the creation of the service, then on the execution which might happen on another timeline when the job is unqueued. We have to revalidate the parameters then. Imagine the author has lost their rights: in that case we should not run the job.